package.json

```diff
@@ -8,23 +8,30 @@
     "json",
     "bemjson"
   ],
-  "main": "lib/bemReact.js",
+  "main": "lib/index.js",
   "author": "Dmitry Filatov <dfilatov@yandex-team.ru>",
   "peerDependencies": {
     "react": "0.14.x"
   },
   "devDependencies": {
+    "babel-cli": "^6.4.5",
+    "babel-preset-es2015": "^6.3.13",
+    "babel-preset-es2015-loose": "^7.0.0",
```

Comment:
Why do you include both `babel-preset-es2015` and `babel-preset-es2015-loose` presets?